osgar/drivers/rosmsg.py

```diff
@@ -346,8 +346,14 @@ def slot_request_origin(self, timestamp, data):
         cmd = b'request_origin'
         self.bus.publish('cmd_vel', cmd)
 
+    def send_filename(self):
+        filename = self.bus.logger.filename  # deep hack
+        cmd = b'file ' + bytes(filename, encoding='ascii')
+        self.bus.publish('cmd_vel', cmd)
```

Comment:
Isn't this even bigger hack? Sending filename to output called `cmd_vel`?

Reply:
Yes, you are right :( ... it was originally only commands with velocity, but then I needed to request origin position and stdout ... and now file. So the output of "rosmsg" should have different name - it is the channel to C++ code ...  suggestion for new name? Thanks m.